src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -1190,3 +1194,88 @@ def run_datamover(self, test_desc=None,
                 self.fail("stderr expected {}: {}".format(s, test_desc))
 
         return result
+
+    def run_dm_activities_with_ior(self, tool, create_dataset=False):
+        """Generic method to perform varios datamover activities
+           using ior
+        Args:
+            tool(str): specify the tool name to be used
+            create_dataset(bool): boolean to create initial set of
+                                  data using ior. Defaults to False.
+        """
+        # Set the tool to use
+        self.set_tool(tool)
+
+        if create_dataset:
+            # create initial datasets
+            pool1 = self.create_pool()
+            cont1 = self.create_cont(pool1, oclass=self.ior_cmd.dfs_oclass.value)
+
+            # update and run ior on cont1
+            self.run_ior_with_params(
+                "DAOS", self.ior_cmd.test_file.value,
+                pool1, cont1)
```

Comment:
I don't think it's necessary to specify the oclass on the container if we also specify the same oclass with IOR. Is this "just in case"?

Reply:
This is actually not being used here but it is from the subsequent work I am doing for additional tests where I am using preexisting containers as part of separate test case.